### DISCUSSION TITLE
Grafana Dashboard

### DISCUSSION THREAD

**Discussion by duculete** [jblance/mpp-solar#338] [Ideas]
Hi,

I am sharing a Grafana dashboard based on the metrics:

ac_input_frequency                          float
ac_input_voltage                            float
ac_output_active_power                      float
ac_output_apparent_power                    float
ac_output_frequency                         float
ac_output_load                              float
ac_output_voltage                           float
battery_capacity                            float
battery_charging_current                    float
battery_discharge_current                   float
battery_voltage                             float
battery_voltage_from_scc                    float
bus_voltage                                 float
error                                       string
inverter_heat_sink_temperature              float
is_ac_charging_on                           float
is_battery_voltage_to_steady_while_charging float
is_charging_on                              float
is_charging_to_float                        float
is_configuration_changed                    float
is_load_on                                  float
is_reserved                                 float
is_sbu_priority_version_added               float
is_scc_charging_on                          float
is_scc_firmware_updated                     float
is_switched_on                              float
pv_input_current_for_battery                float
pv_input_power                              float
pv_input_voltage                            float
rsv1                                        float
rsv2                                        float
validity_check                              string

![Screenshot 2023-04-20 at 11 30 30](https://user-images.githubusercontent.com/20344201/233309388-c8e6e458-05df-4c7a-8c3a-aeee76a8b3d8.png)

Remove .txt and go to Grafana and import it. You should know how to add Influxdb as datasource. Db name in influx is mpp-solar/mppsolar.
[MPP-Grafana.json.txt](https://github.com/jblance/mpp-solar/files/11284159/MPP-Grafana.json.txt)

**Comment by Saentist** [comment]
![dashboard](https://github.com/TaiyoMatsumoto/VoltronicAxpertSolarMonitorBMS/raw/main/assets/dashboard1.png)

[json file](https://github.com/TaiyoMatsumoto/VoltronicAxpertSolarMonitorBMS/blob/main/grafana/AXPERT.json)
